helm-component/README.md

```diff
@@ -1,151 +1,9 @@
-   - the **Branch** should be `init`.
-   - the **Tag/Commit** should be `HEAD`.
-   - the **Source format** field should **unstructured**.
-   - the **Policy directory** field should be `helm-component/manifests`.
-
-### Using gcloud
-
-You can also configure the Git repository information in a config-management.yaml file and use a gcloud command to apply the file.
-
-1.  Create a file named config-management.yaml and copy the following YAML file into it:
-    ```yaml
-    # config-management.yaml
-    
-    apiVersion: configmanagement.gke.io/v1
-    kind: ConfigManagement
-    metadata:
-     name: config-management
-    spec:
-     sourceFormat: unstructured
-     git:
-       syncRepo: https://github.com/GoogleCloudPlatform/anthos-config-management-samples/
-       syncBranch: init
-       secretType: none
-       policyDir: helm-component/manifests
-    ```
-1.  Apply the config-management.yaml file:
-    ```console
-    gcloud alpha container hub config-management apply \
-        --membership=CLUSTER_NAME \
-        --config=CONFIG_YAML_PATH \
-        --project=PROJECT_ID
-    ```
-
-   Replace the following:
-   - CLUSTER_NAME: the name of the registered cluster that you want to apply this configuration to
-   - CONFIG_YAML_PATH: the path to your config-management.yaml file
-   - PROJECT_ID: your project ID
-
-## Verifying the installation
-
-### Using GCP Console
-1. In the Cloud Console, go to the [Anthos Config Management](https://console.cloud.google.com/anthos/config_management) page.
-1. View the **Status** column. A successful installation has a status of `Synced`.
-
-### Using gcloud
-Run the following command to get the status
-```console
-gcloud alpha container hub config-management status --project=PROJECT_ID
-```
-Replace `PROJECT_ID` with your project's ID.
-
-A successful installation has a status of `SYNCED`.
-
-### Using nomos
-Run the following command to get the status
-```console
-nomos status
-```
-
-You can also check if the helm component is successfully installed by running
-```console
-kubectl get all -n cert-manager
-```
-
-## Uninstalling the component
-In order to help prevent accidental deletion, Config Sync does not allow you to remove all namespaces or
-cluster-scoped resources in a single commit.
-Follow the instructions to gracefully uninstall the component and remove the namespace in separate commits.
-- Remove the cert-manager component.
-  ```console
-  git rm -rf manifests/cert-manager && git commit -m "uninstall cert-manager" && git push origin init
-  ````
-- Delete the cert-manager namespace.
-  ```console
-  git rm manifests/namespace-cert-manager.yaml && git commit -m "remove the cert-manager namespace" && git push origin init
-  ````
-- Verify the namespace does not exist.
-  ```console
-  Kubectl get namespace cert-namespace
-  ```
+This example provides two different ways of rendering and syncing your configurations
+- manual rendering:
```

Comment:
Link to the manual rendering folder or README

Reply:
DONE